CMakeLists.txt

```diff
@@ -98,6 +99,10 @@ if(MUJOCO_ENABLE_AVX_INTRINSICS)
   target_compile_definitions(mujoco PUBLIC mjUSEPLATFORMSIMD)
 endif()
 
+if (MUJOCO_BUILD_SIMULATE_VR)
+  target_compile_definitions(mujoco PUBLIC mjBUILDSIMULATEVR)
```

Comment:
Does this affect MuJoCo itself, or only simulate? If it's the latter, than the compiler definition should go there instead.